ssc/cmod_windpower.cpp

```diff
@@ -616,6 +622,11 @@ void cm_windpower::exec()
 	assign("cutoff_losses", var_data((ssc_number_t)((withoutCutOffLosses - annual) / withoutCutOffLosses)));
 	assign("annual_gross_energy", annual_gross);
 
+    assign("lat", wdprov->lat);
+    assign("lon", wdprov->lon);
+    assign("elev", wdprov->elev);
+    assign("year", wdprov->year);
```

Comment:
The default wind - single owner year value is 1900. Will that be confusing to people?

Reply:
The default cases use "representative files" that have no year value in the header, hence the default 1900. When you download a wind resource file from the WIND Toolkit, the data has a more meaningful year value, which I think might be useful to some users to identify the data (especially SDK/PySAM).

I'm not sure why windpower reads the year value from the wind resource file because it does not seem to be used anywhere in the code. Note that it is required in [Line 111](https://github.com/NREL/ssc/blob/e0297ed5166a90bd46202097ceeedf11ca9b974f/ssc/cmod_windpower.cpp#L111).